### DISCUSSION TITLE
Allow players to collapse each mod catergory

### DISCUSSION THREAD

**Discussion by Gababyte** [ppy/osu#25027] [Ideas]
lazer's mod select screen is pretty cramped, in my opinion. There's little to no proper space for anyone to read the description of each mod.

Sure, you _can_ increase the UI, but doesn't that just, ruin it's purpose?

I suggest to let players collapse each mod category into it's title box (or whatever it's called.)

Not only will this provide more space for the mod selection screen, but it can also:

- Allow players to focus on specific mod categories
- Make the mod selection screen a little less overwhelming
- Allow each category to have it's own little description at the bottom of the title box, just like the little description written on the mods themselves

There are probably more benefits to doing so, but these are the benefits I can think of so far.
![lazer mod select](https://github.com/ppy/osu/assets/64835428/95ed65ae-4910-4e0b-a4a2-9b1dd969b23e)

**Comment by peppy** [comment]
The better solution here is making the descriptions word wrap or just shorter.

**Reply by Gababyte** [reply]
That can solve the issue for the descriptions, but I still think the screen itself is still pretty overwhelming, because you're granted with a ton of mod options, compared to stable.
